setup.cfg

```diff
@@ -0,0 +1,12 @@
+
+# See the docstring in versioneer.py for instructions. Note that you must
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = compliance_checker/_version.py
+versionfile_build = compliance_checker/_version.py
+tag_prefix = v
```

Comment:
@ocefpaf all of the previous releases don't have the v prefix. So moving forward I'll need to tag using v.

Reply:
You don't have to use the v, I do recommend it though, makes the releases more readable.